### ISSUE TITLE
Add CMIP6_3hr table

### ISSUE THREAD

**Issue by YanchunHe** [NorESMhub/noresm2cmor#41]
### The Following fields are now supported:
Fields in 3hr-averaged output (<strike>cam.h3</strike>cam.h4,clm.h2)
```
'mrro      ','QRUNOFF
'mrsos     ','SOILWATER_10CM
'tslsi     ','TSA
```
Fields in 3hr-instantly output (<strike>cam.h4</strike>cam.h5, clm.h3):
```
all not available
```
### The Following fields are not available in the model output:
```
'uas       ','UAS
'vas       ','VAS
'clt       ','CLDTOT
'hfls      ','LHFLX
'hfss      ','SHFLX
'pr        ','PRECT
'prc       ','PRECC
'prsn      ','PRECSC+PRECSL
'rlds      ','FLDS
'rldscs    ','FLDSC
'rlus      ','FLDS+FLNS
'rsds      ','FSDS
'rsdscs    ','FSDSC
'rsdsdiff  ','SOLLD+SOLSD
'rsus      ','FSDS-FSNS
'rsuscs    ','FSDSC-FSNSC
```

### Sample cmorized fields
under /scratch/yanchun/cmorout/NorESM2-LM/historical/v20190815/3hr
```
mrro_3hr_NorESM2-LM_historical_r1i1p1f1_gn_201001010130-201001312230.nc
mrsos_3hr_NorESM2-LM_historical_r1i1p1f1_gn_201001010130-201001312230.nc
tslsi_3hr_NorESM2-LM_historical_r1i1p1f1_gn_201001010130-201001312230.nc
```

### Commit
ca5b932
47a304d

UPDATED: 29 Oct 2019, corrected file tags, so that availability of cmorized files are also changed.

**Comment by IngoBethke** [comment]
Good job. Are uas and vas not available because they are not requested or because we forgot to activate them?

**Comment by YanchunHe** [comment]
They are of course requested by CMIP6, and are put by someone the corresponding model names in the dreq google sheet. I don't know why, but they are not in the output of high-frequency historical run at least.

I will later send this Issue to relevant people to inspect if the above-mentioned information and cmorized data looks OK (they all passed PrePARE quality-check). Thanks!

**Comment by OskarMETNO** [comment]
The CMOR names uas and vas most likely correspond to NorESM variables U10 and V10.

**Comment by YanchunHe** [comment]
I see you have updated these following fields in the google sheet:

6hrPlev | 6hr | atmos | sfcWind | sqrt(U10^2+V10^2)
-- | -- | -- | -- | --
6hrPlevPt | 6hrPt | atmos | sfcWind | sqrt(U10^2+V10^2)
Amon | mon | atmos | sfcWind | sqrt(U10^2+V10^2)
Amon | mon | atmos | uas | U10
Amon | mon | atmos | vas | V10
CF3hr | 3hrPt | atmos | sfcWind | sqrt(U10^2+V10^2)
CFsubhr | subhrPt | atmos | sfcWind | sqrt(U10^2+V10^2)
E3hr | 3hr | atmos | sfcWind | sqrt(U10^2+V10^2)
day | day | atmos | sfcWind | sqrt(U10^2+V10^2)
day | day | atmos | sfcWindmax | X: sqrt(U10^2+V10^2)

But it looks to me this is not correct.
U10 is the full wind speed instead of wind component, so U10 should correspond to sfcWind.
UAS,VAS are not available, however, as I can see in the historical runs.

**Comment by YanchunHe** [comment]
And, `sfcWindmax` is the daily maximum wind speed, which is likely not possible to calculate from daily-averaged u/v wind. It should be a model output, otherwise, can not derive it from post-processing, if I understand correctly.

**Comment by OskarMETNO** [comment]
You are right - it seems U10 is the full wind speed and not the component. I'm sorry, I guess I was confused because for U200 and U850 (and V200 and V850) they are indeed the components.
It is apparently the same also in CAM, though I'm a bit surprised because it's quite unintuitive to use U10 for the wind speed and not the zonal wind.

For the CMIP variables uas and vas we should be able to use the NorESM variables US and VS, although they are staggered.

Regarding sfcWindmax I agree that it should ideally be a model output (preserving the time-step maximum value), but if we do not have that (or do we? I haven't found one) maybe we should still consider using the maximum value of U10?

**Comment by lisesg** [comment]
For the PAMIP experiments, the maximum of daily U10 is included in the h1 files.

**Comment by oyvindseland** [comment]
Hi

Sorry for not noting this thread before 
As far as I can see there may have bee a mix-up between average and instantaneous fields

cam.h2=6 hourly average
cam.h3=6 hourly instantaneous
cam.h4=3 hourly average
cam.h5=3 hourly instantaneous

The highest frequency of U10 is 3 hourly instantaneous (i.e. cam.h5)

**Comment by YanchunHe** [comment]
Thanks for the comments!

Some variables are required by different frequencies, for example, sfcWind is required by quite many tables (frequencies), please update the google sheet file of corresponding table.

Please note, if you changed/corrected in the entries (rows) that have already been cmorzied (indicated in column K) in the google sheet file, please send me an email to notify this, otherwise, I may not notice your changes (because I usually filter those entries that have been cmorized).

I will then create a GitHub issue to keep track of these changes. Or you can directly create an GitHub issues with the template "Add/update CMIP6_table xxx" if you don't find if they have already been created, and I will followup.

**Comment by oyvindseland** [comment]
The staggered wind is not output at any frequency as far as I can see.

In NorESM1 even 10 meter wind was missing so the surface wind was given as U and V from the lowest model layer.  This values is btw also given out at higher frequencies.

**Comment by oyvindseland** [comment]
Hi Is it possible to add more and cmorize more 3 hourly output data for NorESM2-MM historical and scenarios (h4) 
The only exist from 1950 onwards though.
Specifically I have received a question about 3hourly precipiation data and that exists in the output files.

From
 ncdump -h NHISTfrc2_f09_tn14_20191025.cam.h4.1951-01-01-10800.nc
...
        float PRECT(time, lat, lon) ;
                PRECT:units = "m/s" ;
                PRECT:long_name = "Total (convective and large-scale) precipitation rate (liq + ice)" ;
                PRECT:cell_methods = "time: mean" ;
...

**Comment by YanchunHe** [comment]
The precipitation was in the 3hourly instant files (h5) instead of 3 hourly average (h4) files in the variable namelist file of the cmor tool.

I have set it as the 3hourly average output in h4 file.

Those will be cmorized soon.

**Comment by YanchunHe** [comment]
I don't remember why this was reopened Jan. 2024, but I will close it now.
